### PR TITLE
Add VMAF-integrated dashjs for TV devices

### PR DESCRIPTION
**Pull request by minhkstn**

### CODE REVIEW

File: client/player/dashjsVmafTV/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+<div id='trace' style="height:640px;width:480px;border:1px solid #ccc;font:16px/26px Georgia, Garamond;overflow:auto;"></div>
+
+<script type="text/javascript">
+    // source https://streaminglearningcenter.com/encoding/best-practices-for-netflixs-vmaf-metric.html
+    // source https://eyevinntechnology.medium.com/keep-an-eye-on-the-video-quality-b9bcb58dd5a1
+    const minAcceptableVMAF = 20;
+    const maxAcceptableVMAF = 93;
+    
+    const context = this.context
```
Comment (tbabak) [inline client/player/dashjsVmafTV/index.html:24]: I don't see this being used anywhere

File: client/player/dashjsVmafTV/index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    })
+    
+    // limit bitrate based on VMAF
+    var filterCapabilities = function (representation) {
+        if (representation.mimeType === "video/mp4" && representation.vmaf > 0) {
+            return representation.vmaf >= minAcceptableVMAF && representation.vmaf <= maxAcceptableVMAF;
+        }
+        return true;
+    }
+    player.registerCustomCapabilitiesFilter(filterCapabilities) ;
```
Comment (tbabak) [inline client/player/dashjsVmafTV/index.html:70]: space before the semicolumn might cause problems in some situations

**Review by tbabak — CHANGES_REQUESTED** [review]
@minhkstn i didnt check the content of [4.3.0.debug.dash.js](https://github.com/cd-athena/CMCD-CAdViSE/pull/17/files#diff-03d6ecd97ce2132f5ae5c37ad7bf0893d536887e7ea25c992cc556a82619faa3)
hope that you have tested it.
other than that only some minor changes requested.

File: client/player/dashjsVmafTV/index.html (hunk 1 above)
Comment (tbabak) [reply client/player/dashjsVmafTV/index.html:24]: no sorry I meant the `context` variable

Comment (minhkstn) [reply client/player/dashjsVmafTV/index.html]: ah got it. I deleted it.

File: client/player/dashjsVmafTV/index.html (hunk 2 above)
Comment (minhkstn) [reply client/player/dashjsVmafTV/index.html]: Updated the code.

**Review by tbabak — APPROVED** [review]

**Pull request merged**